apps/meteor/client/views/room/contextualBar/UserInfo/UserInfo.js

```diff
@@ -61,7 +61,7 @@ function UserInfo({
 				{actions && <InfoPanel.Section>{actions}</InfoPanel.Section>}
 
 				<InfoPanel.Section>
-					<InfoPanel.Title title={(showRealNames && name) || username || name} icon={status} />
+					<InfoPanel.Title title={showRealNames && name ? name : username} icon={status} />
```

Comment:
maybe use `getUserDisplayName` here? This way we can unify where we make this assertions.

Reply:
THAT'S WHAT I'M SAYING, DUDE!